Translations/pl.po

```diff
@@ -401,14 +401,14 @@ msgstr ""
 
 #: Source/DiabloUI/hero/selhero.cpp:282 Source/DiabloUI/hero/selhero.cpp:285
 msgid "Enter Name"
-msgstr "Wpisz Imię"
+msgstr "Wpisz Nazwę"
 
 #: Source/DiabloUI/hero/selhero.cpp:314
 msgid ""
 "Invalid name. A name cannot contain spaces, reserved characters, or reserved "
 "words.\n"
 msgstr ""
-"Niepoprawne imię. Nazwa nie może zawierać spacji, znaków specjalnych i "
+"Niepoprawna nazwa. Nie może ona zawierać spacji, znaków specjalnych i "
```

Comment:
Ditto about "nazwa" vs. "imię" when related to a character.

Reply:
We agree with you.